Shoot/README.md

```diff
@@ -122,6 +124,13 @@ $ open platforms/ios/Shoot.xcodeproj/
 
 #### Android setup
 
+```bash
+$ cordova platform add android@master --usegit
```

Comment:
why would we want to install the master version of the android platform and not the latest version?